### ISSUE TITLE
pre weights can not use !!!

### ISSUE THREAD

**Issue by violet2422** [Lornatang/WassersteinGAN_DIV-PyTorch#3]
`    model = torch.hub.load("Lornatang/WassersteinGAN_DIV-PyTorch", args.arch, progress=True, pretrained=True,
                           verbose=False)`
Downloading: "https://github.com/Lornatang/WassersteinGAN_DIV-PyTorch/releases/download/0.1.0/Wasserstein_DIV_lsun-700f9016.pth"  url is False?

**Comment by Lornatang** [comment]
Yes, the code is still being updated at the moment, the link will be updated later
